pkg/kev/config/svcextensions.go

```diff
@@ -107,29 +107,27 @@ func DefaultSvcK8sConfig() SvcK8sConfig {
 	}
 }
 
-type svcK8sConfigOptions struct {
-	requireExtensions bool
-	disableValidation bool
+type extensionOptions struct {
+	skipValidation bool
 }
 
-// SvcK8sConfigOption will modify parsing behaviour of the x-k8s extension.
-type SvcK8sConfigOption func(*svcK8sConfigOptions)
+// K8sExtensionOption will modify parsing behaviour of the k8s extension.
+type K8sExtensionOption func(*extensionOptions)
 
-func DisableValidation() SvcK8sConfigOption {
-	return func(options *svcK8sConfigOptions) {
-		options.disableValidation = true
-	}
-}
-
-// RequireExtensions will ensure that x-k8s is present and that it is validated.
-func RequireExtensions() SvcK8sConfigOption {
-	return func(options *svcK8sConfigOptions) {
-		options.requireExtensions = true
+// SkipValidation skips validation when parsing a k8s extension from a service.
+func SkipValidation() K8sExtensionOption {
```

Comment:
Tiny one: should we move the shared bits to a separate file?

Reply:
I was considering introducing a `types.go` file in that pkg in the next ticket.

Will leave for now :).